### PR TITLE
Example: with Freactal

### PR DESCRIPTION
**Pull request by srigi**
I created example employing [FormidableLabs/freactal](https://github.com/FormidableLabs/freactal) - a state management library that allows state composability and reduces boilerplate code to absolute minimum.

Demo app use power of universal rendering - first, page is rendered server-side (data fetched from API by the server), then, following page(s) of the dataset are fetched by client from the same endpoint.

See [readme](https://github.com/srigi/next.js/blob/example/freactal/examples/with-freactal/README.md) for more details.

### CODE REVIEW

**Review by timneutkens — APPROVED** [review]
Very nice!

File: examples/with-freactal/.babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (timneutkens) [inline examples/with-freactal/.babelrc]: This can be removed. Since there is no special babel configuration

Comment (srigi) [reply examples/with-freactal/.babelrc]: @timneutkens You're right, this is uneeded, I removed it from the example.

**Review by timneutkens — APPROVED** [review]
Thanks!

**Pull request merged**